docusaurus.config.js

```diff
@@ -209,7 +209,28 @@ module.exports = {
       {
         to: '/reference/compose/',
         from: ['/reference/stacks/']
-      }]
+      },
+      {
+        to: '/administration/dashboard/',
+        from: ['/enterprise/administration/dashboard/']
+      },
+      {
+        to: '/administration/cleanup/',
+        from: ['/enterprise/administration/cleanup/']
+      },
+      {
+        to: '/administration/private-repositories/github-app/',
+        from: ['/enterprise/administration/private-repositories/github-app/']
```

Comment:
In SaaS, we are creating the GitHub App by ourselves. The environment is already integrated with a GitHub App from start. Should we move the GitHub App configuration doc under Self-Hosted?

Reply:
Yeah, if it's only for Self-Hosted, I'll put the configuration part under self-hosted